pages/home.jsx

```diff
@@ -88,6 +87,11 @@ var LatestPosts = React.createClass({
 });
 
 var BlogSection = React.createClass({
+  getDefaultProps: function() {
+    return {
+      loadBlogPosts: loadBlogPosts
```

Comment:
Basically this is for the view's unit tests, allowing us to easily substitute the real blog post loader (well, it'll *eventually* be real) with a fake one.